docs/source/libraries/keras.rst

```diff
@@ -42,6 +42,12 @@ Important arguments to :func:`eli5.explain_prediction` for ``Model`` and ``Seque
     
     - Check ``model.input_shape`` to confirm the required dimensions of the input tensor.
 
+* ``image`` Pillow image, corresponds to doc input.
+
+    - **Must be passed for image explanations.**
+            
+    - **Must have mode "RGBA".**
```

Comment:
I think it would be more convenient for the user to accept any PIL image and then convert it to RGBA